File: app/src/main/java/com/android/record/MyApplication.java
package com.android.record;

import android.app.Application;

import com.android.record.db.DaoManager;

/**
 * @author chen_kai
 * @date：2021-07-15
 * @desc：程序入口
 */
public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        DaoManager.getInstance().init(this);
    }

}

File: app/src/main/java/com/android/record/MyApplication.kt
package com.android.record

import android.app.Application
import com.android.record.db.DaoManager

/**
 * @author chen_kai
 * @date：2021-07-15
 * @desc：程序入口
 */
class MyApplication : Application() {

    companion object {
        lateinit var instance: MyApplication
    }

    override fun onCreate() {
        super.onCreate()
        instance = this
        DaoManager.getInstance().init(this)
    }

}